src/index.js

```diff
@@ -5,9 +5,34 @@
 const fs = require('fs');
 const feedQueue = require('./feed-queue');
 const feedWorker = require('./feed-worker');
+const parentLogger = require('../src/lib/logger');
+
+const log = parentLogger.child({ module: 'module-name' });
```

Comment:
`'module-name'` needs to be a name.  In this case, you could use something like `'main'`.

Reply:
Sure